src/shogun/labels/DenseLabels.h

```diff
@@ -31,7 +32,7 @@ namespace shogun
 	 * label
 	 * (vectors).
 	 */
-	class DenseLabels : public Labels
+	class DenseLabels : public Labels, public SGContainerIteratorBase<DenseLabels, float64_t>
```

Comment:
same question about multi inheritance, although this should be hidden somehow (if i get it right) that we never actually expose DenseLabels directly to swig